Build error in xrdp-neutrinordp.c

I get a build error for xrdp-neutrinordp.c in xrdp v0.9.14.
Only when Neutrinordp is enabled.

```
 ./configure --enable-neutrinordp
make

Making all in neutrinordp
make[2]: Entering directory `/tmp/test/xrdp/neutrinordp'
  CC     xrdp-neutrinordp.lo
xrdp-neutrinordp.c: In function 'lfreerdp_polygon_sc':
xrdp-neutrinordp.c:1397:12: error: unused variable 'npoints' [-Werror=unused-variable]
     int i, npoints;
            ^
xrdp-neutrinordp.c: In function 'lfreerdp_synchronize':
xrdp-neutrinordp.c:1451:17: error: variable 'mod' set but not used [-Werror=unused-but-set-variable]
     struct mod *mod;
                 ^
cc1: all warnings being treated as errors
make[2]: *** [xrdp-neutrinordp.lo] Error 1
make[2]: Leaving directory `/tmp/test/xrdp/neutrinordp'
make[1]: *** [all-recursive] Error 1
make[1]: Leaving directory `/tmp/test/xrdp'
make: *** [all] Error 2
```

v0.9.13 also shows the error, but the build is complete.

--- a/neutrinordp/xrdp-neutrinordp.c
+++ b/neutrinordp/xrdp-neutrinordp.c
@@ -1394,7 +1394,7 @@ static void
 lfreerdp_polygon_sc(rdpContext *context, POLYGON_SC_ORDER *polygon_sc)
 {
     struct mod *mod;
-    int i, npoints;
+    int i;
     struct {
         short x, y;
     } points[4];
@@ -1448,8 +1448,11 @@ lfreerdp_polygon_sc(rdpContext *context, POLYGON_SC_ORDER *polygon_sc)
 static void
 lfreerdp_synchronize(rdpContext *context)
 {
+    /* Uncomment these two lines when needed */
+#if 0
     struct mod *mod;
     mod = ((struct mod_context *)context)->modi;
+#endif
     LLOGLN(12, ("lfreerdp_synchronize received - not handled"));
 }
 

